### PR TITLE
Interactive menu using javascript

### PR DESCRIPTION
**Pull request by gabyse1**
## This branch aims to add interactive features to the menu button.

- Create a menu section that is made up of a close button and a menu navigator.
- Add styles to the markup created in the above step.
- Add javascript functions and listener events to handle the menu interaction.
- Add id attributes to each section in order to link them with their respective menu link.

### CODE REVIEW

File: scripts/main.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+function toggleMenu() {
```
Comment (carlos-ssh) [inline scripts/main.js:1]: **Kindly install `ESLint` in your linters project. You can have a copy of the setting files** [here](https://github.com/microverseinc/linters-config/tree/master/html-css-js).
_After install, run your linters locally and ensure to no have issues after all working well without errors you can request another code review._

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
# 🦺 🚧 🚥 CHANGES REQUIRED 🚥 🚧 🦺
#

Hi @gabyse1 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._ 


changesrequested
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
```
Comment (carlos-ssh) [inline .eslintrc.json:1]: **Your project stills missing `ESLint` in your `Pull Request`.**
_Keep on mind if you added already `ESLint` setup files it's required to run the installations following the steps described on_ [this section](https://github.com/microverseinc/linters-config/tree/master/html-css-js).

Your Linters should appear like this:
Screen Shot 2021-12-13 at 20 57 27

#
Add your `ESLint` to your list:

Screen Shot 2021-12-13 at 20 57 53

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
# 🦺 🚧 🚥 CHANGES REQUIRED 🚥 🚧 🦺
#

Hi @gabyse1, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._ 


changesrequested
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by alonsofalconi — APPROVED** [review]
Hi @gabyse1 and @JohnFTitor,

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**